tests/test_foreign.py

```diff
@@ -339,9 +399,6 @@ def testPickle(self):
             self.assertTrue(isinstance(g, Graph))
             self.assertTrue(g.vcount() == 3 and g.ecount() == 1 and not g.is_directed())
             g.write_pickle(tmpfname)
-
-    
-
```

Comment:
Some separation would be great between test cases. I agree that three lines is maybe too much, but no newlines at all may be too little :)

Reply:
Ha, oh sorry, must have removed those lines accidentally :)